io.github.endless_sky.endless_sky.plugins.endless_sky_high_dpi.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>io.github.endless_sky.endless_sky.plugins.endless_sky_high_dpi</id>
+  <extends>io.github.endless_sky.endless_sky</extends>
+  <name>Endless Sky High DPI</name>
+  <summary>High resolution graphics for Endless Sky</summary>
+  <developer_name>Michael Zahniser</developer_name>
+  <project_license>GPL-3.0-or-later AND GPL-2.0-or-later AND CC-BY-SA-4.0 AND CC-BY-SA-3.0 AND CC-BY-4.0 AND CC-BY-3.0 AND CC-BY-2.0 AND CC0-1.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <description>
+    <p>This plugin provides double-resolution graphics for Endless Sky. These graphics will be used only if you have a high-DPI monitor or your zoom factor (in the preferences) is set higher than 100%.</p>
+  </description>
+  <url type="bugtracker">https://github.com/endless-sky/endless-sky-high-dpi/issues</url>
+  <url type="homepage">https://github.com/endless-sky/endless-sky-high-dpi/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://private-user-images.githubusercontent.com/20605679/246690308-87ae00d1-43a8-4cf5-82a7-c50719dcacc5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMTM4NjYsIm5iZiI6MTcxMDExMzU2NiwicGF0aCI6Ii8yMDYwNTY3OS8yNDY2OTAzMDgtODdhZTAwZDEtNDNhOC00Y2Y1LTgyYTctYzUwNzE5ZGNhY2M1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDIzMzI0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkYWRmMjMyMWFkODRjY2UzMTYyYzE3ZWM4NjM2YjE2NTI3ODM2NzM4NGI1MjcwYWI3OGM0YmQ5YjZiMmE1MzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lPA-v6Gpt5mUCUR4YhhqrWLnCQ_D7ooKySmEcvCBlTk</image>
```

Comment:
This screenshot url doesn't work, extensions don't need one, so checks are skipped. If this was an actual app, it'd fail validation.

Reply:
Oh it seems to be a private URL for just my account. I didn't realize.